src/test/rmc/DynTrait/main.rs

```diff
@@ -34,10 +29,10 @@ fn random_animal(random_number: i64) -> Box<dyn Animal> {
 }
 
 fn main() {
-    let random_number = rmc::nondet();
+    let random_number = rmc::any();
     let animal = random_animal(random_number);
     let s = animal.noise();
-    if random_number < 5 {
+    if (random_number < 5) {
```

Comment:
Changed by mistake?

Reply:
Probably! Some of these changes are from a couple months ago.